.travis.yml

```diff
@@ -1,17 +1,28 @@
-dist: bionic
+dist: focal
 language: php
+cache:
+  directories:
+  - $HOME/.composer
+  - $HOME/.npm
+env:
+  - SITEID=docstore
 
 php:
   - 8.0
 
-services:
-  - docker
-
-# Make sure we have a recent version of docker.
+# Make sure we have a recent version of docker-compose.
 addons:
   apt:
     packages:
-      - docker-ce
+      - docker-compose
+
+mysql:
+  database: drupal
+  username: root
+  encoding: utf8
```

Comment:
If we're testing in docker, this isn't needed since we have our own mysql docker image in the test stack.